include/SFML/System/FileInputStream.hpp

```diff
@@ -111,36 +111,36 @@ class SFML_SYSTEM_API FileInputStream : public InputStream
     /// \param data Buffer where to copy the read data
     /// \param size Desired number of bytes to read
     ///
-    /// \return The number of bytes actually read, or -1 on error
+    /// \return The number of bytes actually read, or std::nullopt on error
```

Comment:
```suggestion
    /// \return The number of bytes actually read, or `std::nullopt` on error
```

Doxygen should support inline code blocks
https://www.doxygen.nl/manual/markdown.html#md_codespan

Reply:
Is it out of scope to fix that in other places where `std::nullopt` is used in a comment without surrounding backticks?

Reply:
I wouldn't care that much, but probably best to leave it for a separate PR. I'm quite sure I've inadvertently used single quotes before...